codemon/CodemonInit.py

```diff
@@ -6,8 +6,26 @@ def write_to_file(filename, text, contestName=None):
   full_filename = os.path.join(os.getcwd(), filename)
   if contestName is not None:
     full_filename = os.path.join(os.getcwd(), contestName, filename)
-  with open(full_filename, 'w+') as f:
-    f.write(text)
+
+    os.makedirs(full_filename)
+    
+    cppFile = full_filename + '/' + filename + '.cpp'
+    f = open(cppFile,"w+")
+    template = template_cpp()
+    f.write(template)
+
+    inpFile = full_filename + '/' + filename + '.in'
+    f = open(inpFile,"w+")
+    outFile = full_filename + '/' + filename + '.op'
+    f = open(outFile,"w+")
+
+    cppTest = full_filename + '/' + 'test_case.cpp'
+    f = open(cppTest,"w+")
+    inpTest = full_filename + '/' + 'test_case.in'
+    f = open(inpTest,"w+")
+    outTest = full_filename + '/' + 'test_case.out'
+    f = open(outTest,"w+")

```

Comment:
What are the purposes of `test_case.in`, `test_case.out` ? Which file will contain the generated test cases ? How will it be compared to the current output ?

Reply:
test_case.in will contain the generated test cases.
test_case.out will contain the output to the generated test cases.
The current output will be compared to the test_case.out.

Reply:
Is it the correct way to do so?